src/components/input/_base.scss

```diff
@@ -28,9 +28,18 @@ $transitions-label-height: 1.2rem;
     cursor: pointer;
     z-index: $z-2;
 
+    &::placeholder {
+        color: transparent;
+        transition: color 250ms;
+    }
+
     &:focus {
         outline: none;
         cursor: text;
+
+        &::placeholder{
```

Comment:
Some reason i can't edit my comments on a phone but what I'd rather place this under is the active selector class